toolset/utils/docker_helper.py

```diff
@@ -182,9 +190,14 @@ def watch_container(docker_container, docker_file):
                 'soft': 99
             }]
 
+            docker_cmd = ''
+            if hasattr(test, 'docker_cmd'):
+                docker_cmd = test.docker_cmd
+
             container = self.server.containers.run(
                 "techempower/tfb.test.%s" % test.name,
                 name=name,
+                command=docker_cmd,
```

Comment:
I suspect it does, but this will override the `CMD` present in their dockerfile, right?

Reply:
Right. And if it’s an empty string it will keep the CMD in the dockerfile.